test/inductor/test_flex_attention.py

```diff
@@ -6860,6 +6863,120 @@ def test_flex_attention_with_dynamic_max_autotune(self, device):
     def test_flex_attention_with_dynamic_max_autotune_graph_partition(self, device):
         self._test_flex_attention_with_dynamic_max_autotune(device)
 
+    @skip_on_cpu
+    def test_flex_attention_logging(self, device):
```

Comment:
There is a tmpdir pytest fixture you can use, I think just adding arg the test, tmpdir or tmp_dir might trigger it

Reply:
I dont think pytorch supports this fixture